client/state/selectors/get-inline-support-article-action-label.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * @param {object} state Global app state
+ * @returns {object} ...
+ */
+export default state => state?.inlineSupportArticle?.actionLabel ?? translate( 'Visit article' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Visit Article' )` *ES Score: 11*